sdk/voiceservices/Azure.ResourceManager.VoiceServices/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Release History
+
+## 1.0.0 (2023-02-16)
```

Comment:
Change the version to `1.0.0-beta.1` as any new stable should have beta ahead for 1 month

Reply:
Ok, I wasn't sure because our service is already GA, but I've now corrected, sorry about that